gradle/build-scan.gradle

```diff
@@ -18,8 +18,8 @@ buildScan {
         tag 'CI'
         tag System.getenv('JOB_NAME')
         link 'Jenkins Build', System.getenv('BUILD_URL')
-        link 'GCP Upload',
-                "https://console.cloud.google.com/storage/elasticsearch-ci-artifacts/jobs/${jobName}/${buildNumber}.tar.bz2"
+        link 'GCP Logs',
```

Comment:
I'm inclined to call this something more like "Build Logs" or "Additional build logs". The "GCP" bit here is irrelevant. If these were stored in S3 it would make no functional difference.